app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -271,6 +275,29 @@ def open_claims
     end
   end
 
+  # Check both Benefits Claim service and Caseflow Appeals status APIs for open 030 or 040
+  # Offramps EP 400 Merge process if any are found, or if anything fails
+  def open_claim_review?
+    open_claim_review = open_claims.any? do |claim|
+      CLAIM_REVIEW_BASE_CODES.include?(claim['base_end_product_code']) && OPEN_STATUSES.include?(claim['status'])
+    end
+    if open_claim_review
+      StatsD.increment("#{EP_MERGE_STATSD_KEY_PREFIX}.open_claim_review")
+      return true
+    end
+
+    ssn = User.find(user_uuid)&.ssn
+    ssn ||= auth_headers['va_eauth_pnid'] if auth_headers['va_eauth_pnidtype'] == 'SSN'
+    decision_reviews = Caseflow::Service.new.get_appeals(OpenStruct.new({ ssn: })).body['data']
+    StatsD.increment("#{EP_MERGE_STATSD_KEY_PREFIX}.caseflow_api_called")
+    decision_reviews.any? do |review|
+      CLAIM_REVIEW_TYPES.include?(review['type']) && review['attributes']['active']
+    end
+  rescue => e
+    Rails.logger.error('EP Merge failed open claim review check', backtrace: e.backtrace)
```

Comment:
Will the SSN be in `e.backtrace`?

Reply:
The backtrace only includes method names, and the SSN is a temporary local variable in the current method so it wouldn't be included.